src/main/java/no/nav/tag/finnkandidatapi/kandidat/KandidatMapper.java

```diff
@@ -16,6 +16,9 @@
 
     @Override
     public Kandidat mapRow(ResultSet rs, int i) throws SQLException {
+        if (rs.getBoolean(SLETTET)) {
+            return null;
+        }
 
         LocalDateTime sistEndret = rs.getTimestamp(REGISTRERINGSTIDSPUNKT) == null ? null : rs.getTimestamp(REGISTRERINGSTIDSPUNKT).toLocalDateTime();
         ArbeidstidBehov arbeidstidBehov = rs.getString(ARBEIDSTID_BEHOV) == null ? null : ArbeidstidBehov.valueOf(rs.getString(ARBEIDSTID_BEHOV));
```

Comment:
Minor case: har vi en maks lengde for linjer i kildekoden? med f.eks maks på 120 skal vi kunne unngå horizontal scrolling her på GitHub.